### ISSUE TITLE
Create application flow chart

### ISSUE THREAD

**Issue by GlennKintscher** [KatastrophenTeam/BreaktimeSnack#5]
Create a flow chart that shows the functionality of the application

**Comment by BodooKing** [comment]
Awaiting Closing the #3

**Comment by GlennKintscher** [comment]
@BodooKing Time to continue

**Comment by BodooKing** [comment]
https://docs.google.com/document/d/1wCLHZyxAqrO2DXb4WPv0Q3la8FSBIDGqxGhAE66zziQ/edit?usp=sharing

**Comment by GlennKintscher** [comment]
@BodooKing Could you summarize it in English as well? Just the most important things
